### PR TITLE
Created a route to actus table and removed the actus_text and actus_img routes 

### PR DESCRIPTION
**Pull request by Hacene010**

### CODE REVIEW

**Review by GeorgesFORTRY — APPROVED** [review]
ok

**Review by GeorgesFORTRY — APPROVED** [review]
ok

**Review by GeorgesFORTRY — APPROVED** [review]
ok

File: routes/actus.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const actusRouter = require('express').Router();
+const connection = require('../db-config');
+
+actusRouter.get('/', (req, res) => {
+  connection.query(
+    'SELECT title, img_src, img_alt, link, content, date_time, is_active FROM actus ORDER BY date_time DESC',
+    (err, rows) => {
+      if (err) {
+        console.log(err);
```
Comment (LoicBrassart) [inline routes/actus.js]: Nope

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
No console logs

**Review by GeorgesFORTRY — APPROVED** [review]
ok

**Review by LoicBrassart — APPROVED** [review]
NmE!

**Pull request merged**